py/mpconfig.h

```diff
@@ -490,6 +490,11 @@
 #define MICROPY_LONGINT_IMPL (MICROPY_LONGINT_IMPL_NONE)
 #endif
 
+// Support for calls to buildin.pow() with 3 integer arguments
+#ifndef MICROPY_INT_POW3
+#define MICROPY_INT_POW3 (1)
```

Comment:
Actually, the option should be named MICROPY_PY_BUILTINS_POW3 per our conventions, sorry for not spotting this before...